consensus/src/liveness/proposal_generator_test.rs

```diff
@@ -120,66 +117,3 @@ async fn test_old_proposal_generation() {
         .err();
     assert!(proposal_err.is_some());
 }
-
-#[tokio::test]
-async fn test_empty_proposal_after_reconfiguration() {
```

Comment:
@zekun000, for you to review.

Reply:
why this test failed?

Reply:
It inserts reconfiguration block using the hacky test helper function.

Reply:
status here?